test/performance/README.md

```diff
@@ -2,8 +2,9 @@
 
 ## Configuring your cluster to run a benchmark
 
-1. Create a namespace or use an existing namespace. Each namespace can be
-   configured with a single benchmark.
+1. Create a namespace `perf-eventing` if not exists. To use a different
+   namespace, pleaes replace all the namespaces in
+   all bash commands and yaml configuration files with your choice.
 
 1. Install Knative eventing by following the steps in
    https://github.com/knative/eventing/blob/2c6bf0526634804b7ebeee686445901440cc8edd/test/performance/performance-tests.sh#L31
```

Comment:
Is it still relevant to recommend following this script which is not up-to-date?

Reply:
Good point, I personally didn't check this at all. I simply install knative eventing myself. Should we just change it to "install Knative eventing and components under test such as MT broker"?

Reply:
I think that would be a good idea, and link to DEVELOPMENT.md